service/history/queues/queue_immediate.go

```diff
@@ -157,7 +157,9 @@ func (p *immediateQueue) processEventLoop() {
 		case <-p.shutdownCh:
 			return
 		case <-p.notifyCh:
-			p.processNewRange()
+			if err := p.processNewRange(); err != nil {
+				p.logger.Error("error processing new range", tag.Error(err))
```

Comment:
If you want to follow logger message convention, it should be 
```suggestion
				p.logger.Error("Unable to process new range.", tag.Error(err))
```

Reply:
Done